db/migrate/20190902103812_create_posts.rb

```diff
@@ -0,0 +1,12 @@
+# frozen_string_literal: true
+
+class CreatePosts < ActiveRecord::Migration[5.2]
```

Comment:
You can choose to ignore a comment like the one below from stickler, but I'd suggest that you address it as it will only improve how you write your ruby/rails code.

A simple comment like `# This class does ...`

However, these are rails generated files so you also add it to ignore list in your .stickler.yml. You may also use rubocop in local IDE to correct this linting issue seeing as it's based on a rails generated file.

Otherwise, for linting issues like length of a line of code etc, I suggest you fix manually as to allow yourself the opportunity to practice writing clean ruby code.